signalfx-org-metrics/docs/sf.org.numResourceMonitored.md

```diff
@@ -0,0 +1,12 @@
+---
+title: sf.org.numResourceMonitored
+brief: Number of host based resources monitored
+metric_type: counter
+---
+### sf.org.num.numResourceMonitored
+
+Number of hosts/containers that are being monitored.
```

Comment:
hosts/containers ==> hosts or containers

Add sentence "The `resourceType` dimension indicates whether the value represents hosts or containers.